### ISSUE TITLE
Support signing transaction without web3.js

### ISSUE THREAD

**Issue by GabrielePicco** [coral-xyz/backpack#1638]
# Problem description

The current [SignTransaction](https://github.com/coral-xyz/backpack/blob/master/packages/provider-core/src/provider-solana-xnft.ts#L109) (Solana implementation) only accepts a @solana/web3.js Transaction object. In the [Solana.Unity-SDK](https://github.com/garbles-labs/Solana.Unity-SDK) we would like to sign transactions that are byte[] or base58 encoded string, so that we can avoid importing Web3.js in games compiled to WebGL with the SDK.

**Comment by armaniferrante** [comment]
Has this been resolved with a workaround @GabrielePicco, since I see you have a successful xNFT running with unity?

**Comment by GabrielePicco** [comment]
@armaniferrante yes it currently works with a workaround, we inject web3js in a custom WebGL template and serialise/deserialise the transaction. If you happen to implement bytes signature in the future, we can streamline the integration.

**Comment by armaniferrante** [comment]
Does the new solana wallet standard support this now? https://github.com/coral-xyz/backpack/blob/master/packages/wallet-standard/src/wallet.ts#L234
